src/common/utils/validator.js

```diff
@@ -240,14 +240,38 @@ function currentOrPastDate(dObj) {
  * @param {Object} dObjh Date - can be js Date object or the {d:,m:,y} type object that is used in the UI
  * * @returns {Bool} Date is within acceptable range
  */
-function dateNotTooFarInFuture(dObj) {
+function dateNotMoreThanMonthInFuture(dObj) {
   const now = new Date();
   const nextMonth = new Date(now.getFullYear(), now.getMonth() + 1, now.getDate());
   const providedDate = getDateFromDynamicInput(dObj);
 
   return Boolean(providedDate && providedDate <= nextMonth);
 }
 
+function dateNotMoreThanTwoDaysInFuture(providedDate) {
+  const now = new Date();
+  const TWO_DAYS = 2 * 24 * 60 * 60 * 1000;
+  const maxDepartureDate = new Date(now.getTime() + TWO_DAYS);
+
+  if (!(providedDate instanceof Date)) {
+    return false;
+  }
+
+  return Boolean(providedDate && providedDate.getTime() <= maxDepartureDate.getTime());
+}
+
+function isTwoHoursPriorDeparture(providedDate) {  
+  const TWO_HOURS = 60 * 60 * 1000 * 2;
```

Comment:
I think maybe TWO_HOURS_MILLISECONDS would be a more appropriate constant name.  And on line 264 maybe, start with **2 *** like in the **dateNotMoreThanTwoDaysInFuture** function.

Reply:
Changes made.